src/assets/index.js

```diff
@@ -1,21 +1,21 @@
 import logo from "./images/logo/shuttlemotion.png";
-import WorlcupInterviewThumbnail from "./images/video-thumbnail/BWF-Players-interview.png";
-import SindhuInterviewThumbnail from "./images/video-thumbnail/PV-Sindhu-Interview.png";
-import BadmintonTricksThumbnail from "./images/video-thumbnail/10-Amazing-Badminton-Trickshots.png";
-import BiomechanicsTricksThumbnail from "./images/video-thumbnail/Shots-biomechanics.png";
-import GoldMedalMatchThumbnail from "./images/video-thumbnail/Badminton-Gold Medal-Match.png";
+import WorlcupInterviewThumbnail from "./images/video-thumbnail/BWF-Players-interview.jpg";
+import SindhuInterviewThumbnail from "./images/video-thumbnail/PV-Sindhu-Interview.jpg";
+import BadmintonTricksThumbnail from "./images/video-thumbnail/10-Amazing-Badminton-Trickshots.jpg";
+import BiomechanicsTricksThumbnail from "./images/video-thumbnail/Shots-biomechanics.jpg";
+import GoldMedalMatchThumbnail from "./images/video-thumbnail/Badminton-Gold Medal-Match.jpg";
 import SaniaMatchThumbnail from "./images/video-thumbnail/Sania-Nehwal-Match.png";
-import LinDanChongThumbnail from "./images/video-thumbnail/Lin-Dan -vs-Lee-Chong-Wei.png";
+import LinDanChongThumbnail from "./images/video-thumbnail/Lin-Dan -vs-Lee-Chong-Wei.jpg";
```

Comment:
One small suggestion for the images would be you can host them on some place I have done it on google drive make it public and use them directly where needed
Helps you in writing less code and also need not to have a assets folder for assets only.

Reply:
Thanks for suggesting Akshat. I am also thinking to host them in some cloud service. Appreciate your feedback and will work on it.

Reply:
Done the changes thanks for the feedback.